PWGLF/DataModel/LFStrangenessTables.h

```diff
@@ -325,10 +328,15 @@ DECLARE_SOA_DYNAMIC_COLUMN(PositiveEta, positiveeta, //! positive daughter eta
                            [](float PxPos, float PyPos, float PzPos) -> float { return RecoDecay::eta(std::array{PxPos, PyPos, PzPos}); });
 DECLARE_SOA_DYNAMIC_COLUMN(PositivePhi, positivephi, //! positive daughter phi
                            [](float PxPos, float PyPos) -> float { return RecoDecay::phi(PxPos, PyPos); });
+
+DECLARE_SOA_DYNAMIC_COLUMN(IsStandardV0, isStandardV0, //! is standard V0
+                           [](uint8_t V0Type) -> bool { return V0Type & (1 << 0); });
+DECLARE_SOA_DYNAMIC_COLUMN(IsPhotonV0, isPhotonV0, //! is photon V0
```

Comment:
I have been thinking about the naming here. This is extremely misleading actually because this flag is only set to true if the photo mass hypothesis _and_ if a TPConly track is one of the prongs. This could potentially lead analyzer to believe that using this flag that they get photons only but in fac they are _also only_ then getting tpconly prongs… a better name might be ‘isPhotonTPConly’ (or something else, I am lacking creativity here).